src/devtools/useAtomsSnapshot.ts

```diff
@@ -1,12 +1,10 @@
 import { useContext, useMemo } from 'react'
-import { SECRET_INTERNAL_getStoreContext as getStoreContext } from 'jotai'
-import type { Atom } from 'jotai'
-
-import type { Scope } from '../core/atom'
+import {
+  SECRET_INTERNAL_getStoreContext as getStoreContext,
+  SECRET_INTERNAL_useMutableSource as useMutableSource,
+} from 'jotai'
 import type { AtomState, State } from '../core/vanilla'
-
-// XXX across bundles, this is actually copying code
-import { useMutableSource } from '../core/useMutableSource'
+import { Atom, Scope } from '../core/atom'
```

Comment:
```suggestion
import type { Atom, Scope } from '../core/atom'
```
We are strict about this. This helps a bit to understand what code is across bundles.

Reply:
That's a mistake, will fix it right away